helpers/google/index.js

```diff
@@ -368,6 +368,12 @@ var execute = function(LocalGoogleConfig, collection, service, callObj, callKey,
     } else if (callObj.nested) {
         parentParams = {auth: callObj.params.auth, parent: callObj.params.parent};
         executor['projects']['locations']['keyRings'][service][callKey](parentParams, LocalGoogleConfig, executorCb);
+    } else if (callObj.params && callObj.location && callObj.postcall && callObj.projectId && callObj.regional) {
+        parentParams = {auth: callObj.params.auth, projectId: callObj.params.project, location: callObj.params.region, [callObj.filterKey[0]]: callObj.params[callObj.filterKey[0]]};
+        executor['projects']['locations'][service][callKey](parentParams, LocalGoogleConfig, executorCb);
+    } else if (callObj.projectId && callObj.regional) {
+        parentParams = {auth: callObj.params.auth, projectId: callObj.params.project, location: callObj.params.region};
+        executor['projects']['locations'][service][callKey](parentParams, LocalGoogleConfig, executorCb);
```

Comment:
@AkhtarAmir  is this needed? I don't see it relating to the call

Reply:
This was in another PR so I added here as well.